R/assignment_linter.R

```diff
@@ -6,6 +6,32 @@
 #'   If `FALSE`, [`<<-`][base::assignOps] and `->>` are not allowed.
 #' @param allow_right_assign Logical, default `FALSE`. If `TRUE`, `->` and `->>` are allowed.
 #' @param allow_trailing Logical, default `TRUE`. If `FALSE` then assignments aren't allowed at end of lines.
+#'
+#' @examples
+#' library(lintr)
+#'
+#' # will produce lints
+#' lint(
```

Comment:
for ones with parameters I think it's more important to showcase their effects

Reply:
I have added examples of parameter usage, except `absolute_path_linter(lax = FALSE)`. Can't come up with a good example for that one.

Reply:
SGTM, I think of this one as more peripheral anyway :sweat_smile: